### PR TITLE
update misc

### PR DESCRIPTION
**Pull request by hooke007**
To update the manual_chs;

For orig manual, I deleted the contents about everything™。

==============

adding support of sub/mks/txt

### CODE REVIEW

File: src/Misc/CorePlayer.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class CorePlayer
         public static string[] VideoTypes { get; set; } = "264 265 asf avc avi avs dav flv h264 h265 hevc m2t m2ts m2v m4v mkv mov mp4 mpeg mpg mpv mts ts vob vpy webm wmv y4m".Split(' ');
         public static string[] AudioTypes { get; set; } = "aac ac3 dts dtshd dtshr dtsma eac3 flac m4a mka mp2 mp3 mpa mpc ogg opus thd thd+ac3 w64 wav".Split(' ');
         public static string[] ImageTypes { get; set; } = { "jpg", "bmp", "png", "gif" };
-        public static string[] SubtitleTypes { get; } = { "srt", "ass", "idx", "sup", "ttxt", "ssa", "smi" };
```
Comment (stax76) [inline src/Misc/CorePlayer.cs:24]: There is a ttxt subtitle format:

https://fileinfo.com/extension/ttxt

**Comment by hooke007** [comment]
I also fixed the wrong default values in the editor.

https://github.com/stax76/mpv.net/blob/42c623c8cae8e13928582c91505d26fb4bdefc60/src/Resources/mpv.conf.txt#L5-L6

**Comment by hooke007** [comment]
`watch-later-options = start` would cause the recording the start time twice, so fix it.

**Review by stax76 — CHANGES_REQUESTED** [review]

**Comment by stax76** [comment]
> watch-later-options = start would cause the recording the start time twice, so fix it.

OK, I understand it now.

File: src/Resources/mpv.conf.txt [hunk 2]
```diff
@@ -8,4 +8,4 @@ osd-duration = 2000
 osd-playing-msg = '${filename}'
 script-opts = osc-scalewindowed=1.5,osc-hidetimeout=2000,console-scale=1
 screenshot-directory = '~~desktop/'
-watch-later-options = start
+watch-later-options =
```
[the remark this answers is not in the document]
Comment (hooke007) [reply src/Resources/mpv.conf.txt:11]: start will cause the issue here
https://github.com/mpv-player/mpv/issues/9174

**Comment by hooke007** [comment]
>as default in the mpv manual.

I know it, but you set the different value in the initial conf

**Comment by hooke007** [comment]
Snipaste_2021-09-02_19-46-49
This can avoid the extra output in console when users drag the window. And it would not affect the other functions.

**Comment by stax76** [comment]
You can put `MBTN_LEFT  ignore` in input.conf, I don't know why it didn't make it into the defaults of input.conf.

**Comment by stax76** [comment]
Empty watch-later-options is not compatible with the conf editor, the related code is complex, for now I work around it setting the default to:

watch-later-options = mute

Not sure if this creates a new problem.

**Comment by hooke007** [comment]
> Not sure if this creates a new problem.

I have a better idea `watch-later-options = vid`
It usually have no side effect.

**Pull request merged**